client/src/components/BookList/BookListTile.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const BookListTile = ({ bookList, user }) => {
+  const divStyle = {
+    backgroundImage: `url(${bookList.imageUrl})`,
+    backgroundRepeat: "no-repeat",
+    backgroundPosition: "center",
+    backgroundSize: "cover",
+  };
```

Comment:
this is fine for now, but is considered "inline styling". ideally we put this in CSS files